AppCenterDistribute/AppCenterDistributeTests/MSDistributeCheckForUpdateTests.m

```diff
@@ -200,8 +200,8 @@ - (void)testCheckForUpdateGetsLatestReleaseEvenThoughAutomaticUpdateIsDisabled {
 
   // Ingestion mock.
   __block id ingestionMock = OCMClassMock([MSDistributeIngestion class]);
-  __block void (^handler)(NSString *callId, NSHTTPURLResponse *_Nullable response, NSData *_Nullable data, NSError *_Nullable error);
   OCMStub([ingestionMock checkForPublicUpdateWithQueryStrings:OCMOCK_ANY completionHandler:OCMOCK_ANY]).andDo(^(NSInvocation *invocation) {
+    void (^handler)(NSString *callId, NSHTTPURLResponse *_Nullable response, NSData *_Nullable data, NSError *_Nullable error);
```

Comment:
@MatkovIvan This needs to be `__unsafe_unretained`. `retainArguments` doesn't help from avoiding crashes.
https://stackoverflow.com/questions/13268502/exc-bad-access-when-accessing-parameters-in-anddo-of-ocmock

Reply:
```suggestion
    __unsafe_unretained  void (^handler)(NSString *callId, NSHTTPURLResponse *_Nullable response, NSData *_Nullable data, NSError *_Nullable error);
```